pkg/monitortestlibrary/pathologicaleventlibrary/duplicated_event_patterns.go

```diff
@@ -2,419 +2,668 @@ package pathologicaleventlibrary
+		})
+	*/
+
+	// various DeploymentConfig tests trigger this by cancelling multiple rollouts
+	registry.AddPathologicalEventMatcherOrDie(&SimplePathologicalEventMatcher{
+		name:               "DeploymentAwaitingCancellation",
+		messageReasonRegex: regexp.MustCompile(`^DeploymentAwaitingCancellation$`),
+		messageHumanRegex:  regexp.MustCompile(`Deployment of version [0-9]+ awaiting cancellation of older running deployments`),
+	})
 
 	// If image pulls in e2e namespaces fail catastrophically we'd expect them to lead to test failures
 	// We are deliberately not ignoring image pull failures for core component namespaces
-	regexp.MustCompile(`ns/e2e-.* reason/BackOff Back-off pulling image`),
-
-	// promtail crashlooping as its being started by sideloading manifests.  per @vrutkovs
-	regexp.MustCompile("ns/openshift-e2e-loki pod/loki-promtail.*Readiness probe"),
-
-	// Related to known bug below, but we do not need to report on loki: https://bugzilla.redhat.com/show_bug.cgi?id=1986370
-	regexp.MustCompile("ns/openshift-e2e-loki pod/loki-promtail.*reason/NetworkNotReady"),
-
-	// kube-apiserver guard probe failing due to kube-apiserver operands getting rolled out
+	registry.AddPathologicalEventMatcherOrDie(&SimplePathologicalEventMatcher{
+		name: "E2EImagePullBackOff",
+		locatorKeyRegexes: map[monitorapi.LocatorKey]*regexp.Regexp{
+			monitorapi.LocatorNamespaceKey: regexp.MustCompile(`^e2e-.*`),
+		},
+		messageReasonRegex: regexp.MustCompile(`^BackOff$`),
+		messageHumanRegex:  regexp.MustCompile(`Back-off pulling image`),
+	})
+
+	// Several allowances were related to Loki, I think we can generally ignore any repeating event
+	// from the Loki NS, this should not fail tests.
+	registry.AddPathologicalEventMatcherOrDie(&SimplePathologicalEventMatcher{
+		name: "E2ELoki",
+		locatorKeyRegexes: map[monitorapi.LocatorKey]*regexp.Regexp{
+			monitorapi.LocatorNamespaceKey: regexp.MustCompile(`^openshift-e2e-loki$`),
+		},
+	})
+
+	// kube apiserver, controller-manager and scheduler guard pod probes can fail due to operands getting rolled out
 	// multiple times during the bootstrapping phase of a cluster installation
-	regexp.MustCompile("ns/openshift-kube-apiserver pod/kube-apiserver-guard.*ProbeError Readiness probe error"),
-	// the same thing happens for kube-controller-manager and kube-scheduler
-	regexp.MustCompile("ns/openshift-kube-controller-manager pod/kube-controller-manager-guard.*ProbeError Readiness probe error"),
-	regexp.MustCompile("ns/openshift-kube-scheduler pod/kube-scheduler-guard.*ProbeError Readiness probe error"),
+
+	registry.AddPathologicalEventMatcherOrDie(&SimplePathologicalEventMatcher{
+		name: "KubeAPIReadinessProbeError",
+		locatorKeyRegexes: map[monitorapi.LocatorKey]*regexp.Regexp{
+			monitorapi.LocatorNamespaceKey: regexp.MustCompile(`openshift-kube-*`),
+			monitorapi.LocatorPodKey:       regexp.MustCompile(`^kube.*guard.*`),
+		},
+		messageReasonRegex: regexp.MustCompile(`^ProbeError$`),
```

Comment:
I'm guessing you know this is the right `reason` since I don't see any `reason/...` on lines 109-112.

Reply:
looking at master, it seems to be the right value.